slackbot/dispatcher.py

```diff
@@ -177,3 +185,7 @@ def channel(self):
     @property
     def body(self):
         return self._body
+
+    def docs_reply(self):
+        reply = ['    • `{0}` {1}'.format(v.__name__, v.__doc__ or "") for p, v in iteritems(self._plugins.commands['respond_to'])]
```

Comment:
well, I did to be able to display the docs of the plugins whenever you want as of [here](https://github.com/iomedhealth/santiago/blob/master/plugins/hello.py#L45)